### ISSUE TITLE
Make "field value" accessible to get/setProperty mocking

### ISSUE THREAD

**Issue by oleksiyp** [oleksiyp/mockk#71]
### Expected Behavior

```
Something like that should be possible:
        val mock = spyk(Team(), recordPrivateCalls = true)

        every { mock.person } answers { fieldValue as String + "A" }
        every { mock.person = any() } answers { fieldValue = value as String + "B" }
        every { mock getProperty "person" } answers { fieldValue as String + "A" }
        every { mock setProperty "person" value any() } answers { fieldValue = value as String + "B" }
```

**Comment by oleksiyp** [comment]
Released 1.7.17. Should be fixed there http://mockk.io/#property-backing-fields

**Comment by erikhuizinga** [comment]
It seems I can't get this feature to work. I tried to implement an example based on the code from the readme, but it results in `io.mockk.MockKException: Missing calls inside every { ... } block.`

Example:

```kotlin
class DynamicCallsTest {
	@Test
	fun example() {
		val mock = spyk(Car(), recordPrivateCalls = true)

		val expectedSpeed = 33
		val expectedAcceleration = 4
		val a = "left"
		val b = "rear"
		val expectedOpenDoor = "OK"

		every { mock getProperty "speed" } returns expectedSpeed
		every { mock setProperty "acceleration" value less(5) } just Runs
		every { mock invoke "openDoor" withArguments listOf(a, b) } returns expectedOpenDoor

		mock.set(10, expectedAcceleration)
		assertEquals("$expectedSpeed, $expectedAcceleration", mock.get())
		assertEquals(expectedOpenDoor, mock.doOpenDoor(a, b))

		verify { mock getProperty "speed" }
		verify { mock setProperty "acceleration" value less(5) }
		verify { mock invoke "openDoor" withArguments listOf(a, b) }
	}
}

class Car(private var speed: Int = 0, private var acceleration: Int = 0) {
	fun set(speed: Int, acceleration: Int) {
		this.speed = speed
		this.acceleration = acceleration
	}

	fun get() = "$speed, $acceleration"

	fun doOpenDoor(a: String, b: String) = openDoor(a, b)

	private fun openDoor(a: String, b: String) = "$a, $b"
}
```
